examples/helm/templates/_helpers.tpl

```diff
@@ -91,7 +91,7 @@ Create the name of the service account to use
 
 {{- define "jitsi-meet.publicURL" -}}
 {{- if .Values.publicURL }}
```

Comment:
We might change the condition here so the `publicURL` is expected as a fallback if the ingress is not enabled:

```handlebars
{{- if .Values.web.ingress.enabled -}}
```